### PR TITLE
re-add ipv6 support for network resources

### PR DESCRIPTION
**Pull request by zaibon**

### CODE REVIEW

File: pkg/network/ndmz/ndmz.go [hunk 1]
```diff
@@ -111,23 +147,43 @@ func Delete() error {
...
 
-		var (
-			master netlink.Link
-			err    error
-		)
-
+		// find which interface to use as master for the macvlan
 		if namespace.Exists(types.PublicNamespace) {
-			master, err = getPublicIface()
+			pubNS, err := namespace.GetByName(types.PublicNamespace)
```
Comment (LeeSmet) [inline pkg/network/ndmz/ndmz.go:155]: this err is not checked

**Review by LeeSmet — CHANGES_REQUESTED** [review]
LGTM except 1 small remark

File: pkg/network/ndmz/ndmz.go (hunk 1 above)
Comment (delandtj) [reply pkg/network/ndmz/ndmz.go:155]: well yea.. it Exists(), so could as well be _

Comment (LeeSmet) [reply pkg/network/ndmz/ndmz.go:155]: Not sure about that, the implementation of GetByName seems to also open a file, which could error even if it exists, as far as I know.

**Pull request merged**